### PR TITLE
Feat/wasm3bench

### PR DESCRIPTION
**Pull request by tolauwae**

### CODE REVIEW

File: benchmarks/flash_and_check.py [hunk 1]
```diff
@@ -52,6 +59,9 @@ def await_output(s: serial.Serial, target: str):
         await_output(serial, "START\n")
         startTime = time.monotonic()
         print("START found, waiting for DONE", file=sys.stderr)
-        await_output(serial, "DONE\n")
-        endTime = time.monotonic()
-        print(endTime - startTime)
+        success = await_output(serial, "DONE\n", failure="Guru Meditation Error")
```
Comment (beardhatcode) [inline benchmarks/flash_and_check.py:62]: Guru Meditation Error?

**Review by beardhatcode — APPROVED** [review]

File: benchmarks/flash_and_check.py (hunk 1 above)
Comment (tolauwae) [reply benchmarks/flash_and_check.py:62]: when wasm3 runs out of stack space

Comment (beardhatcode) [reply benchmarks/flash_and_check.py:62]: Is that a string Wasm3 returns in this case?

Comment (tolauwae) [reply benchmarks/flash_and_check.py:62]: No that's the standard error on the ESP32

**Pull request merged**